agent/handlers/agentapi/taskprotection/v1/handlers/handlers.go

```diff
@@ -38,7 +38,11 @@ import (
 )
 
 const (
-	ExpectedProtectionResponseLength = 1
+	ExpectedProtectionResponseLength           = 1
+	ProtectionFailureReasonMissing             = "MISSING"
```

Comment:
Is it correct to understand that one of such "failure"s will only be returned by ECS FE API in case of partial successes? Is "partial success" applicable to how ECS Agent will make the request? 

I was under impression that ECS Agent will only make request for one task at a time and will never batch requests (because we use task role credentials to sign). So my understanding was that Agent will never get failure, only exception from FE. A non-null failure seems a code bug somewhere...

Please let me know if there's some gap in my understanding.

Reply:
"failure"s will only be returned by ECS FE API to "support" partial success. In a GetTaskProtection case, even if one out of one request in batch request failed, it will still be a failure instead of exception.